packages/compass-aggregations/src/modules/pipeline-builder/text-editor-pipeline.ts

```diff
@@ -134,7 +147,6 @@ const reducer: Reducer<TextEditorState> = (state = INITIAL_STATE, action) => {
     return {
       ...state,
       serverError: null,
-      syntaxErrors: [],
```

Comment:
Related to the comment above fetching of preview shouldn't affect syntax errors that are based on the pipeline source